packages/flutter_tools/pubspec.yaml

```diff
@@ -104,4 +104,4 @@ dartdoc:
   # Exclude this package from the hosted API docs.
   nodoc: true
 
-# PUBSPEC CHECKSUM: 8411
+# PUBSPEC CHECKSUM: d112
```

Comment:
normally, this checksum should only ever be written by `flutter update-packages --force-upgrade`. It basically hashes the entire file (except for the last line), basically to prevent people from manually updating package versions (which I did). We have a test that will verify that the contents of every pubspec.yaml in the repo matches the checksum at the bottom.

I made this PR by manually updating the dwds version, running the test locally, and updating the checksum to match the failure message.

The algorithm for computing the checksum lives https://github.com/flutter/flutter/blob/master/packages/flutter_tools/lib/src/commands/update_packages.dart#L1530

Reply:
Awesome! Thank you, this process seems similar to how we solve license failure issues in the engine.

Reply:
LOL, now that you mention it I bet the same person wrote both :)